tests/testthat/test-matrix_form.R

```diff
@@ -1,10 +1,12 @@
 test_that("matrix_form works with and without indentation", {
+  require("dplyr", quietly = TRUE)
```

Comment:
Should it be after the skip_if_not_installed?

Reply:
should be ok :)